spec/changelog.yml

```diff
@@ -2,6 +2,11 @@
 ## This file documents changes in the package specification. It is NOT a package specification file.
 ## Newer entries go at the bottom of each in-development version.
 ##
+- version: 2.11.0-next
+  changes:
+  - description: Add owner.type field to indicate package support level
+    type: enhancement
+    link: https://github.com/elastic/package-spec/pull/568
 - version: 2.10.1-next
```

Comment:
Please, remove this block as it won't be needed.

Reply:
Not quite sure I understand, do you want me to put the changelog entry under `2.10.1-next` instead?

Reply:
No, we have to remove at some point the `2.10.1-next` block, but no worries, we can also remove it later.

Reply:
I'll put it under `2.10.1-next` so at least the entry is in there, but we can remove/rearrange later like you say.